single.php

```diff
@@ -32,17 +32,19 @@
 									else:
 										echo_documents_cover();
 									endif;
+
+									if(get_the_content()):
+										echo "<p>".get_the_content()."</p>";
+									endif;
+
+									if(odm_echo_extras()):
+										odm_echo_extras();
+									endif;
+
+									if(echo_downloaded_documents()):
```

Comment:
I understand the problematic but the implementation is not so clean. I am going to merge this for the moment and open a PR with a change suggestion for you to review a bit later today or during the weekend.